src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 145.1.1. [TC-CDOCONC-1.1] Global Attributes with DUT as Server
```

Comment:
It's much too bad that we have to duplicate what amounts to N times the *exact same test* (for N clusters), rather than being able to just use the "by ID" method for both cluster ID, and attribute IDs.

Reply:
Let's punt. I am asking @vivien-apple to see if we can improve this over time and backfill the improvement on these tests.

Reply:
Well, the `cluster` argument in those file is part of the `config` section. So one should already be able to configure the `cluster` used with `./scripts/tests/yaml/chiptool.py tests Test_TC_OO_1_1 --cluster ColorControl` for example.

Now those files has different PICS and so far there is no mechanism to configure the PICS value based on some other values (`cluster` in this case). We could probably extend the PICS support to have a default "context" (e.g "CDOCONC"). It may both allow to configure the context from the command line as well as removing the need to duplicate the context in the PICS section all the time.